tests/integration/test_main.py

```diff
@@ -97,10 +97,18 @@ async def test_handle_interface_deleted(self):
         interface.switch.dpid = "00:00:00:00:00:00:00:01"
         stats_event = KytosEvent(name=event_name,
                                  content={'interface': interface})
+        self.napp.get_intf_usage = MagicMock(return_value="Usage")
```

Comment:
It'd be cool for completeness to assert that `log.info` would be called, and `assert not self.napp._delete_interface.call_count` here too (similar to what you done below, but also mocking `napp._delete_interface`)

Reply:
Added these asserts